sale_delivery_rate/README.rst

```diff
@@ -0,0 +1,75 @@
+.. image:: https://img.shields.io/badge/license-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License: LGPL-3
+
+===================
+Sale Delivery Rates
+===================
+
+This module extends the notion of delivery rate quotes introduced in 
+``stock_picking_rate`` to sale orders. The new quote type is compatible with the 
+``Set price`` button used on sale orders to calculate a delivery price. Also, 
+when a sale order with such rate quotes is confirmed and results in the 
+creation of a stock picking, the picking will be updated with equivalent stock 
+picking delivery quotes.
```

Comment:
I can try rewording this, but I'm not sure I can do much better. Can you try out the sale order -> stock picking flow on Runbot and let me know if it's still unclear?